### PR TITLE
Milestone 8 : Popup Window

### PR DESCRIPTION
**Pull request by Zohra-Neda**
**In this project we :** 

1. Added a fancy popup window section by using DOM Manipulation to enhance the user experience
2.  Made all card sections dynamic to give a fresh and engaging look to the website
3. Created a dynamic project section to showcase the latest and greatest projects in an interactive way
4. Styled the popup for both mobile and desktop devices, ensuring that it looks great on any screen size

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  id: 1,
+  image: 'Img Placeholder.svg',
+  title: 'Multi-Post Stories',
+  paragraph: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+  lang1: 'CSS',
+  lang2: 'HTML',
+  lang3: 'Bootstrap',
+  lang4: 'Ruby',
+  button: 'See Project',
+};
```
Comment (Ibizugbe) [inline script.js:35]: Nice work on implementing the functionalities of the popup👍😉 It works as expected👍. but taking a look at your code, we can make it better (DRY (less repetitive)) and save some lines of code by simply creating just one array for all your projects and your popup too😉 Let's have just one array of objects as required in [this milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) and extract the data for your project section and popup😉 It feels a bit repetitive to have three different arrays for contents that refer to your project section😉

File: script.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+          <p class="popup-p">${popupContent.paragraph}</p>
+          <div class="buttons-container">
+            <button class="popup-btn1" type="button"><a class="popup-links" href="#">${popupContent.button1}</a><span class="fa-sharp fa-solid fa-arrow-up-right-from-square popup-icon"></span></button>
+            <button class="popup-btn2" type="button"><a class="popup-links" href="#">${popupContent.button2}</a><span class="fa fa-github popup-icon" popup-icon"></span></button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>`;
+}
```
Comment (Ibizugbe) [inline script.js:164]: Your popup looks good on mobile😉 But there are a bit of adjustment you can make to ensure you stick to the design😉 Kindly take a look at the comparison below, take note of the spacings, and font properties and make the changes where necessary👍 Don't forget we are still required to match [the design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=CyhOIlQXPS37WqN4-0) in[ this milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md)😉

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|your MOdal|desin modal|

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Zohra-Neda 👋👋👋, 

tryagain


Good job so far!👍👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are properly setup and they are passing👍.
- [x] The functionalities of the popup have been properly implemented👍.
- [x] Github flow practice have been used👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Kindly add contents to your FAQ sections or take it away if you do not have contents for it as this section is optional😉 as this is giving your repository a less professional look🥲
image


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (danmainah) [reply script.js:160]: - [ ] @Zohra-Neda great work implementing the changes requested however as shown below, on a wider screen the tech items seem to move inside the popup, you can add ```align-self : flex-start ``` on the items container.
image

**Review by AbelG101 — APPROVED** [review]
# Hi @Zohra-Neda 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Great UI, on point ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**